dynatrace/api/documents/document/settings/document.go

```diff
@@ -44,6 +45,12 @@ func (me *Document) Schema() map[string]*schema.Schema {
 			Required:         true,
 			ValidateDiagFunc: ValidateMaxLength(200),
 		},
+		"private": {
+			Type:        schema.TypeBool,
+			Description: "Specifies whether the document is private or readable by everybody",
+			Optional:    true,
+			Default:     false,
```

Comment:
Considering documents have been private up to now, wondering if this should be `true` to preserve this for any existing documents, or if we need to migrate state somehow. I think we ultimately want documents to be public by default though, correct?

Reply:
Had similar thoughts. My idea was that we might want to call the attribute `public` and let it default to `false` (essentially because it's more intuitive for users that an absent `boolean` defaults to `false`.
Will reach out to you guys on Monday - most likely I'll come up with fifty other ideas.

Reply:
Thought about Arthurs comment over the weekend. In the end customers expect to see newly provisioned dashboards to show up for other users - not for the service user they're authenticating with using Terraform.
In order words, like Arthur said, we (and customers) want documents to be public by default anyways. I expect that customers will welcome that we're flagging any existing documents as `public`.